langkit/tests/test_themes.py

```diff
@@ -21,7 +19,7 @@ def test_theme(interactions):
     # default input col is "prompt" and output col is "response".
     # since our df has different input col name, let's specify it.
     from langkit import themes
-    schema = DeclarativeSchema(generate_udf_schema())
+    schema = udf_metric_schema()
```

Comment:
If we apply the standard resolver we need to turn off frequent items for the prompt/responses so that we remain privacy preserving (also these are typically longer strings anyway).